docs/guides/utilizing-forms.md

```diff
@@ -52,17 +52,21 @@ To save your form in Camunda Platform, click **File > Save File As...** in the t
 
 ## Connect your form to a BPMN diagram
 
+Next, let's implement a task form into a diagram. In tandem, we can connect your form to a user task or start event.
+
 Click on the **Forms** tab again to view a list of all the forms you've created.
 
-Next, let's implement a task form into a diagram. To connect your form to a user task or start event, take the following steps:
+Take the following steps:
```

Comment:
What I find interesting here is that you assume cloud to be the default (and branch of where Platform differs). I think that could work, if users know about it (upfront). But then again, why not clearly differentiate at the start (cf. my previous examples).

Reply:
Brought the note to the beginning of the section and clarified discrepancies between Camunda Modeler and Cloud Modeler in latest commit. Thoughts?

Reply:
I like what you did here @christinaausley. It's an awkward line to walk, but it fits in this context.